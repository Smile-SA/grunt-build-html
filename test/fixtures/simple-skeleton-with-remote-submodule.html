<%= include('simple-skeleton', {
  title: 'My title',
  body: retrieve('http://www.nantes.fr/sites/nantesfr/home/footerLinks/footer-link.html.ajax')
}) %>