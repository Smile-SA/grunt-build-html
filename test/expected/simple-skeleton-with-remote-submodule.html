<!DOCTYPE html>
<html>
<head>
    <title>My title</title>
</head>
<body>
<ul>
    <li>
        <a href="/home/mentions-legales.html">
            Mentions l√©gales</a></li>
    <li>
        <a href="/home/contact.html">
            Nous contacter</a></li>
    <li>
        <a href="/espace-presse">
            Espace presse</a></li>
    <li>
        <a href="/abonnement-lettre-information">
            Lettre d&#039;information</a></li>
    <li>
        <a href="/home/ville-de-nantes/proximite/relation-aux-usagers/nantes-passion.html">
            Nantes Passion</a></li>
    <li>
        <a href="/home/a-votre-service/fiches-pratiques/mairie/consulter-votre-messagerie-mail.html">
            Mail in Nantes</a></li>
</ul>
</body>
</html>